colabs/analyzeFilmsInImdbDataset.ipynb

```diff
@@ -1,111 +1,113 @@
 {
```

Comment:
Jeżeli już modyfikujemy skrypt ładowania danych, to ja bym przyłożył funkcję do wszystkich interesujących nas kolumn (kolumn do "złanhotenkodowania").

Reply:
celem zadania było zakodowanie kolumny rodzaj_produkcji a poza tym jak narazie tylko ta kolumna nadaje się do użycia onehotencoding